addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -107,7 +112,10 @@ options.registry.gallery = options.Class.extend({
                     this.trigger_up('cover_update');
                 }
             });
-            dialog.on('closed', this, () => resolve());
+            dialog.on('closed', this, () => {
+                this.__imageDialogOpened = false;
+                resolve();
```

Comment:
Without the braces `{}` before, it was actually returning, now it doesn't anymore
```suggestion
                return resolve();
```